zilencer/models.py

```diff
@@ -5,15 +5,18 @@
 import zerver.models
 import datetime
 
+API_KEY_LENGTH = 64
```

Comment:
Yeah, it was like that before but now these are used in two models, RemoteZulipServer and RemoteServerRegistrationStatus.